### ISSUE TITLE
An increase in the number of phrases and remarks for combat units and for peasants.

### ISSUE THREAD

**Issue by Russianstory** [Sh0wdown/UnofficialCrusaderPatch#521]
![aыыыыva](https://user-images.githubusercontent.com/48765277/73666935-9c62ce00-46b4-11ea-956b-04746a97571a.jpg)

Can anyone help me and answer. Is it realistic to increase the number of available phrases for each unit in the game. The fact is that in Stronghold 2 there are excellent phrases and turns in combat units. I think it would be very interesting and entertaining if someone increased (probably this is a matter of programming or scripts) the number of replicas. And I would be able to drag these replicas into the Stronghold Crusader. I want to enrich the vocabulary of phrases. If it works out, I’ll start dragging phrases into the first Stronghold.

**Comment by LordHansCapon** [comment]
@Russianstory Short answer: Yes.

**Long answer:**
Luckily Firefly didnt encode sfx files, and the sfx filenames are hardcoded into the EXE. So we can search for all of the voiceline arrays, and see what accesses them, then inject our own array with more files and reroute these accesses to our new array. Hopefully the game also loads these new files at startup according to these arrays or loads every sound file by default form the folder, or else it will be harder.

![hxd](https://user-images.githubusercontent.com/59705872/73680621-df28a400-46bc-11ea-8499-a58d53ebaab8.PNG)

**Comment by Russianstory** [comment]
@LordHansCapon This is just great. I would be glad if you prepared free slots for replicas. Then I could drag more phrases from other games in the series. I don’t think it should be done =)

**Comment by Russianstory** [comment]
I have already downloaded
Stronghold 2
Stronghold 3
Stronghold legends
I will weed out and group unit phrases. I hope everything works out.

P.S. and even if you can remove all the intro at the beginning of the game when you enter. Why now already in 2020, these video screensavers? Please remove (delete) them, so that after starting, IMMEDIATELY go to the main menu where the knight stands!

**Comment by GRhin** [comment]
Russianstory, can you remember to create new issues for extra ideas? In this case removing the intro video (which I disagree with, a couple clicks and can skip it, but it's part of what crusader is imho). This idea is not related to the original topic and so should go in another ticket so the devs can see it, and others can discuss it.

**Comment by Russianstory** [comment]
Yes, you are probably right. About the video you need to do a separate topic. I just remembered it just wrote.

**Comment by Russianstory** [comment]
@LordHansCapon Can you help me? Can you increase the number of free slots for unit phrases? I already downloaded and checked everything. Phrases are many. If you know how, then please increase their number, and I already do it myself further.

**Comment by Russianstory** [comment]
AND ??

**Comment by Russianstory** [comment]
And if you can please add more slots for all sorts of different sounds of war and battles. Blows, screams, moans of death, shooting, fire, destruction and hitting buildings, etc. Most. And I'm already throwing sounds. Plus of course add slots for phrase units. I would really appreciate it.

**Comment by LordHansCapon** [comment]
I currently don't have time to work on any mods, but I will be notifying you when I'm back.

**Comment by Russianstory** [comment]
@LordHansCapon Friend. I’ll ask you, if you can succeed, make an increase in the number of sounds: groans from pain, shooting, blows from weapons, deadly screams, stone shells entering buildings, fire and other everyday and natural sounds. I will try to enrich the atmosphere with sounds.

**Comment by Russianstory** [comment]
Still relevant on 03/14/2020

**Comment by Russianstory** [comment]
Still relevant on 03/29/2020

**Comment by Lolasik011** [comment]
@Russianstory Напиши ему на прямую в Дискорде ...

**Comment by Lolasik011** [comment]
Там вся актуальная инфа !

**Comment by Russianstory** [comment]
О привет)) вы все туда перебрались отсюда чтоли? ))

**Comment by Lolasik011** [comment]
> О привет)) вы все туда перебрались отсюда чтоли? ))

Нет. Там тоже много инфы.

**Comment by Russianstory** [comment]
как найти вас там?

**Comment by Lolasik011** [comment]
> как найти вас там?

Попроси приглашение у @Krarilotus ...

**Comment by GRhin** [comment]
Краралотус дал инструкции в теме "люди хотят знать". ссылка находится на патче github readme (просто нажмите «код» вверху экрана, чтобы увидеть это) и в вики (нажмите «вики» вверху экрана, чтобы увидеть это)

krarilotus did give instructions in the "people want to know" thread. a link is on the patch github readme (just click 'code' at top of screen to see that) and in the wiki (click 'wiki' at top of screen to see that)

**Comment by Russianstory** [comment]
@Lolasik011 нихрена не нашёл))) где она должна быть))))
